class/defaults.yml

```diff
@@ -24,7 +24,22 @@ parameters:
         operator:
           clusterPoolIPv4MaskSize: "23"
           clusterPoolIPv4PodCIDR: 10.128.0.0/14
-      kubeProxyReplacement: probe
+      kubeProxyReplacement: partial
+      sessionAffinity: true
+      hostServices:
+        enabled: true
+      nodePort:
+        # Explicitly select ensX device as direct routing device for node-port
+        # traffic. If multiple ens* devices are present on a node, the one
+        # with the lowest alphanumerical name is picked.
+        # Adjust this parameter if your nodes don't have host interfaces which
+        # are named `ens.+`.
+        directRoutingDevice: ens+
```

Comment:
Is the `+` special syntax or a typo?

Reply:
Syntax to select any interface which starts with `ens`, cf. https://docs.cilium.io/en/v1.12/gettingstarted/kubeproxy-free/#nodeport-devices-port-and-bind-settings

Reply:
Weird choice, but repeated in the docs, so I guess also not a typo on their side.

Reply:
I've reworded the comment to not add to the confusion. Also, I've seen this syntax before, but I don't remember where.